src/commands/synthetics/interfaces.ts

```diff
@@ -436,14 +436,31 @@ export interface APIHelperConfig {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface SyntheticsCIConfig extends APIHelperConfig {}
 
-export interface RunTestsCommandConfig extends SyntheticsCIConfig {
+// TODO SYNTH-12989: Clean up deprecated `global` in favor of `defaultTestOverrides`
+export interface LegacyRunTestsCommandConfig extends SyntheticsCIConfig {
   configPath: string
   failOnCriticalErrors: boolean
   failOnMissingTests: boolean
   failOnTimeout: boolean
   files: string[]
-  // SYNTH-12989: Clean up deprecated `global` in favor of `defaultTestOverrides`
   global: UserConfigOverride
+  locations: string[]
+  mobileApplicationVersionFilePath?: string
+  pollingTimeout: number
+  publicIds: string[]
+  selectiveRerun: boolean
+  subdomain: string
+  testSearchQuery?: string
+  tunnel: boolean
+  variableStrings: string[]
+}
```

Comment:
Can you change `LegacyRunTestsCommandConfig` to `extends Omit<>` from the other interface?